paasta_tools/setup_kubernetes_cr.py

```diff
@@ -49,11 +49,15 @@
 from paasta_tools.utils import get_git_sha_from_dockerurl
 from paasta_tools.utils import load_all_configs
 from paasta_tools.utils import load_system_paasta_config
+from paasta_tools.vitesscluster_tools import load_vitess_service_instance_configs
 
 
 log = logging.getLogger(__name__)
 
 
+INSTANCE_TYPE_TO_CONFIG_LOADER = {"vitesscluster": load_vitess_service_instance_configs}
+
+
```

Comment:
You shouldn't need to do this here; this script is already using the LONG_RUNNING_INSTANCE_TYPE_HANDLERS from here, which already has config for vitesscluster instance types: https://github.com/Yelp/paasta/blob/master/paasta_tools/cli/utils.py#L697-L699

If the config loader listed there is incorrect, you should update it to your new one in that file rather than adding new logic here.

Reply:
While this is right in some functionality, especially setup-kubernetes-job, when it comes to setup-kubernetes-cr reconciliation step, the config used to call kubectl client to create/update a resource https://github.com/Yelp/paasta/blob/master/paasta_tools/setup_kubernetes_cr.py#L357 uses config generated from https://github.com/Yelp/paasta/blob/master/paasta_tools/utils.py#L3264 which just loads configs from yelpsoa and not those generated in vitesscluster tooling. I kept running into this and have modified it this way.

Lmk if there is something I'm missing here or if this can be done differently